Usage.md

```diff
@@ -50,6 +50,10 @@ $R1: TensorFlow.Tensor<Float> = [[8.0, 16.0], [24.0, 32.0]]
 $R2: TensorFlow.Tensor<Float> = [32.0, 48.0]
 ```
 
+**Note:** If you encounter issues with the REPL crashing,
```

Comment:
Perhaps it is better to just reference https://bugs.swift.org/browse/TF-940 as a known issue so that when it gets fixed it is clear that this section can be deleted?

Reply:
I think that's a good idea! How about this:
```suggestion
    **Note:** using the `TensorFlow` module in the Swift REPL on macOS is known to
    be problematic since Swift for TensorFlow 0.5.

    ```console
    $ swift
    Welcome to Swift version 5.1-dev (LLVM 200186e28b, Swift 1238976565).
    Type :help for assistance.
     1> import TensorFlow
     2> Tensor(1)
    error: Couldn't lookup symbols:
     TensorFlow.TensorHandle.init(copyingFromCTensor: Swift.OpaquePointer) -> TensorFlow.TensorHandle<A>
     TensorFlow.TensorHandle.init(copyingFromCTensor: Swift.OpaquePointer) -> TensorFlow.TensorHandle<A>
    ```

    [TF-940](https://bugs.swift.org/browse/TF-940) tracks this issue.
```

Reply:
My intention with this phrasing was to channel users to the bug for upvoting and questions/comments. Even after this bug is resolved, it's not unreasonable to expect a different bug to crop up that manifests in a similar way. Even if we forget to remove this from the docs, it could be useful as an example of an issue the user could generate.

That being said, it looks like I included the wrong link to the project itself rather than the specific bug. I'm fine with changing the phrasing so that it's more obvious when it can be removed.

Reply:
Your phrasing is good @dan-zheng - I just moved the issue link closer to the Note for more clarity.